executor/executor.go

```diff
@@ -181,9 +185,23 @@ func (executor *Executor) getFunctionEnv(m *metav1.ObjectMeta) (*crd.Environment
 	return env, nil
 }
 
+func dumpStackTrace() {
```

Comment:
this and the signal/notify goroutine are repeated a few times.  Mind moving them to somewhere common and having just a call like `fissionDebug.setupStackTraceHandler()` in StartWhatever() functions?

Reply:
sure. I'll do that.